README.md

```diff
@@ -115,3 +115,78 @@ if (handleStartupEvent()) {
   return;
 }
 ```
+
+## Checking for updates, downloading and installing
```

Comment:
@kevinsawicki yes, it exists for osx, but as for windows, I'm not sure. I had to implement the similar functionality in scope of my application.